Jenkinsfile

```diff
@@ -86,12 +86,19 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             }
 
             // Build all (for opkg)
-            // TODO grab vault creds and mod config to use OPKGSMIME
             if (params.ALL_PACKAGES){
                 echo 'Enabling all user and kernel packages'
                 sh 'echo \'' \
                  + 'CONFIG_ALL=y\n' \
                  + '\' >> .config'
+
+            // Enable opkg signing
+                withCredentials([
+                    [$class: 'FileBinding', credentialsId: 'opkg-build-private-key', variable: PRIVATE_KEY],
```

Comment:
name can be just opkg-build-key
so that BUILD_KEY.pub and BUILD_KEY would be
opkg-build-key.pub (public key) and opkg-build-key (private key)
Both needs to be present in the workspace at the time of building.